README.md

```diff
@@ -24,11 +24,11 @@ It's comprised of two parts:
 
 Refer to the [example](manifests/operator/) for a reference Kubernetes deployment.
 
-Annotate your service accounts and the operator will create the corresponding
-login role and aws secret role in Vault at
-`auth/kubernetes/roles/<prefix>_aws_<namespace>_<name>` and
-`aws/role/<prefix>_aws_<namespace>_<name>` respectively, where `<prefix>` is the
-string supplied with the `-prefix` flag (default: `vkcc`)
+Annotate your Kubernetes ServiceAccounts and the operator will create the
+corresponding login role and AWS secret role in Vault at
+`auth/kubernetes/roles/<vault-role>` and
+`aws/role/<vault-role>` respectively, where `<vault-role>` is the
+string supplied with the `-vault-role` flag.
```

Comment:
This section applies to the operator, not the sidecar. There's no `-vault-role` flag in the operator.